tachyon_core/src/ffi.rs

```diff
@@ -145,8 +146,13 @@ pub unsafe extern "C" fn tachyon_inserter_insert_integer64(
     inserter: *mut Inserter,
     timestamp: Timestamp,
     value: i64,
-) {
-    (*inserter).insert_integer64(timestamp, value);
+) -> u8 {
+    let result = (*inserter).insert_integer64(timestamp, value);
+
+    match result {
+        Ok(_) => 0u8,
+        Err(tachyon_err) => get_error_code(&tachyon_err),
```

Comment:
Can we return the error in an `out` parameter here and change the doc comments so it reflects that?